subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/ivyresolve/strategy/VersionRangeSelector.java

```diff
@@ -260,62 +260,4 @@ public int hashCode() {
         return result;
     }
 
-    public VersionRangeSelector intersect(VersionRangeSelector other) {
```

Comment:
@bigdaz Is there a replacement for this `intersect` method?

I know this was an internal method to Gradle and not designed to be used externally but this was a useful method for Gradle plugin developers. Ideally a public API would be preferable but even knowing where this internal logic got moved would be useful.